lib/tasks/once-off/cumberland_merge.rake

```diff
@@ -0,0 +1,49 @@
+namespace :once_off do
+  desc "Set merged counties to inactive as of 1/Apr/2020, ensure parent_local_authority valid"
+  task cumberland_merge: :environment do
+    LocalAuthority.create!(name: "Cumberland Council", gss: "E06000063", local_custodian_code: "940", snac: "FAKE", slug: "cumberland", tier_id: Tier.unitary)
+    slugs = %i[allerdale carlisle copeland]
+    successor = LocalAuthority.find_by(slug: "cumberland")
+    parent = LocalAuthority.find_by(slug: "cumbria")
+
+    slugs.each do |slug|
+      la = LocalAuthority.find_by(slug:)
+      if la.parent_local_authority != parent
+        puts("Warning: parent local authority for #{slug} is not as expected (cumbria)")
+        next
+      end
+
+      la.active_end_date = Time.zone.parse("01-Apr-2023")
+      la.active_note = "Merged into the unitary authority Cumberland Council on 1 April 2023"
+      la.succeeded_by_local_authority = successor
+      la.save!
+    end
+
+    LocalAuthority.create!(name: "Westmorland and Furness Council", gss: "E06000064", local_custodian_code: "935", snac: "FAKE2", slug: "westmorland-and-furness", tier_id: Tier.unitary)
+    slugs = %i[barrow-in-furness eden south-lakeland]
+    successor = LocalAuthority.find_by(slug: "westmorland-and-furness")
```

Comment:
It's slightly jarring that we're overwriting variable names for different values for code that ultimately does the same thing (think this is an indicator that the code should be dry'd up), e.g we could add a [reusable method](https://kevinjalbert.com/defined_methods-in-rake-tasks-you-re-gonna-have-a-bad-time/) for https://github.com/alphagov/local-links-manager/pull/1057/files#diff-0586a238f2329057f8c253332dcdc9090609abdef16d39795d402d67201e00e2R27-R38, passing in the slug, successor and active note.